artifacts/deployment/01-sample-apiserver-prerequisites.yaml

```diff
@@ -0,0 +1,85 @@
+# The Prerequisites manifest deploys resources needed to run and discover the sample-apiserver.
+# Those resources include the namespace, needed Roles and RoleBindings and a Service.
+---
+# Namespace where the API server and all related resources are deployed.
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: k8s-sample-apiserver
+---
+# ServiceAccount to be used by the k8s-sample-apiserver.
+kind: ServiceAccount
+apiVersion: v1
+metadata:
+  name: apiserver
+  namespace: k8s-sample-apiserver
+---
+# ClusterRoleBinding to allow API delegation.
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: k8s-sample-apiserver:system:auth-delegator
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: system:auth-delegator
+subjects:
+- kind: ServiceAccount
+  name: apiserver
+  namespace: k8s-sample-apiserver
+---
+# RoleBinding used for authentication.
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: wardle-auth-reader
+  namespace: kube-system
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: extension-apiserver-authentication-reader
+subjects:
+- kind: ServiceAccount
+  name: apiserver
+  namespace: k8s-sample-apiserver
+---
+# ClusterRole used to allow API server to use Admission Webhooks.
+kind: ClusterRole
```

Comment:
We should actually be providing this in core kube.  Can you open an issue and lay out the case for an aggregated cluster role for aggregated API servers?

Reply:
An issue in the `kubernetes/kubernetes` repo?